<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserSearchBook</class>
 <widget class="QWidget" name="UserSearchBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查询书籍</string>
  </property>
  <widget class="QPushButton" name="refreshbutton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>刷新列表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="listeverybookbutton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>420</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>列出所有书籍</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="BookTable">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>621</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>书号</string>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>类别</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>书名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>出版社</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>出版年份</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>作者</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>价格</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>总藏书量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>库存量</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="searchbutton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QLabel" name="cnolabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>借书证号 : </string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>370</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="price">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>价格 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="category">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>120</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>类别 : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="press_line">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>220</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="pricebox1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <double>3000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="author">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>作者 :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="yearbox2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>320</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>3000</number>
   </property>
   <property name="value">
    <number>2021</number>
   </property>
  </widget>
  <widget class="QComboBox" name="category_comboBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>计算机科学</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>软件技术</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>汇编语言</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>数字货币与区块链技术</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>航空航天</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>工业技术</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>政治</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>法律</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>New Item</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>哲学</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>经济</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>历史</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>地理</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>金融</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>农业科学</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>环境科学</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>交通运输</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>数学</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>生物</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>化学</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>物理</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>天文学</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>医药学</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>卫生</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>艺术</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>文学</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>体育</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>语言学</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>外语</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>教育</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>军事</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>马克思主义</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>海洋环境</string>
    </property>
   </item>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="year">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出版年份 : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="author_line">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="bno_line">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>70</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="press">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>220</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出版社 :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="yearbox1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>3000</number>
   </property>
   <property name="value">
    <number>1980</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="bookname_line">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>170</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="bookname">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>170</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>书名 :</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="pricebox2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>370</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <double>3000.000000000000000</double>
   </property>
   <property name="value">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="bno">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>70</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>书号 : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>登出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
